ipatests/pytest_plugins/integration/tasks.py

```diff
@@ -729,7 +729,14 @@ def uninstall_master(host, ignore_topology_disconnect=True,
     if ignore_last_of_role and host_domain_level != DOMAIN_LEVEL_0:
         uninstall_cmd.append('--ignore-last-of-role')
 
-    host.run_command(uninstall_cmd)
+    if verbose and host_domain_level != DOMAIN_LEVEL_0:
+        uninstall_cmd.append('-v')
+
+    result = host.run_command(uninstall_cmd)
+    if "Traceback" in result.stdout_text:
+        logger.error('Error: Traceback occurred during uninstallation')
+        assert False
```

Comment:
Would 'assert Traceback not in result.stdout_text be cleaner here? What does the logging buy us?

Reply:
Okay